### PR TITLE
Corrige objetos eventos do conteúdo inicial. 

### LINKED ISSUE #360: Corrigir erro nos testes com o profile initcontent
Está ocorrendo erro nos testes do profile initcontent, quando tentamos criar eventos.

Ver: https://travis-ci.org/plonegovbr/brasil.gov.portal/jobs/251320573

Esse erro só ocorre nos testes. É possível cria um portal normalmente com o ```./bin/instance```

### PR DESCRIPTION
**Pull request by idgserpro**
No plone.app.event existe start e end ao invés de start_date e end_date.

Também é necessário definir o timezone.

Ver:https://github.com/plone/plone.app.event/blob/1.1.5/plone/app/event/dx/behaviors.py#L73-L124

closes #360

### CODE REVIEW

File: src/brasil/gov/portal/tests/test_init_content.py [hunk 1]
```diff
@@ -242,3 +242,34 @@ def test_installed_packages(self):
         for p in SHOW_DEPS:
             self.assertIn(p, qi)
             self.assertIn(p, installed)
+
+    def test_eventos_available(self):
+        """Testa se a pasta Eventos foi criada"""
+        self.assertTrue('eventos' in self.portal.objectIds(),
```
Comment (hvelarde) [inline src/brasil/gov/portal/tests/test_init_content.py]: use `self.assertIn()` instead.

File: src/brasil/gov/portal/tests/test_init_content.py [hunk 2]
```diff
@@ -242,3 +242,34 @@ def test_installed_packages(self):
...
+                        u'Pasta Eventos não disponível')
+        pasta = self.portal['eventos']
+        self.assertEqual(u'Eventos', pasta.title,
```
Comment (hvelarde) [inline src/brasil/gov/portal/tests/test_init_content.py]: one line only, please.

File: src/brasil/gov/portal/tests/test_init_content.py [hunk 3]
```diff
@@ -242,3 +242,34 @@ def test_installed_packages(self):
...
+                    'start': '2017-01-05 11:00:00',
+                    'end': '2017-01-06 12:00:00'},
+                   {'id': 'evento-3',
+                    'start': '2017-01-10 12:00:00',
+                    'end': '2017-01-11 13:00:00'},
+                   ]
+        d_format = '%Y-%m-%d %H:%M:%S'
+        for evento in eventos:
+            obj_evento = pasta[evento['id']]
+            self.assertEqual(evento['start'],
```
Comment (hvelarde) [inline src/brasil/gov/portal/tests/test_init_content.py]: follow coding conventions:
```python
self.assertEqual(
    evento['start'], obj_evento.start.strftime(d_format))
```

File: src/brasil/gov/portal/tests/test_init_content.py [hunk 4]
```diff
@@ -242,3 +242,34 @@ def test_installed_packages(self):
...
+                             obj_evento.start.strftime(d_format))
+            self.assertEqual(evento['end'],
```
Comment (hvelarde) [inline src/brasil/gov/portal/tests/test_init_content.py]: follow coding conventions:
```python
self.assertEqual(
    evento['end'], obj_evento.start.strftime(d_format))
```

File: src/brasil/gov/portal/tests/test_init_content.py [hunk 5]
```diff
@@ -242,3 +242,34 @@ def test_installed_packages(self):
...
+        """Testa se a pasta Eventos foi criada"""
+        self.assertTrue('eventos' in self.portal.objectIds(),
+                        u'Pasta Eventos não disponível')
+        pasta = self.portal['eventos']
+        self.assertEqual(u'Eventos', pasta.title,
+                         u'Título não aplicado')
+
+    def test_eventos_created(self):
+        """Testa se os eventos foram criados corretamente"""
+        pasta = self.portal['eventos']
```
Comment (hvelarde) [inline src/brasil/gov/portal/tests/test_init_content.py]: variable names in Portuguese?

**Comment by idgserpro** [comment]
@hvelarde pronto.

**Pull request merged**